Poppy/app-edge-locate.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "app-edge-locate-0.0.1",
+  "name": "app-edge-locate",
+  "version": "0.0.1",
+  "platform": "complete",
+  "dependencies": [
+    {
+      "name": "app-platform-complete",
+      "version": "0.0.4"
+    },
+    {	
```

Comment:
@OleksiiKuzminov opinions on this?  It's redundant since app-platform-complete depends on app-platform-minimal, but I suppose it can't hurt to explicitly state the dependency on both.  I forget what our position is on this.

Reply:
Yes. it should be just a app-platform-complete